src/lib.rs

```diff
@@ -0,0 +1,3 @@
+extern crate core;
```

Comment:
Do we need `lib` file?

Reply:
As the handlers are under `handlers` folder, if we remove it, it won't be able to find the module.

This line that you mention isn't necessary, I just removed in my last commit.

Reply:
Yes, but adding
`mod handles;`
to main.rs will be enough as we will be building only bin.